README.md

```diff
@@ -24,7 +24,7 @@ Read more on [getting a token](https://www.eventbrite.com/developer/v3/api_overv
 
 ## Contributing
 
-Coming soon...
+Check out our contributing document [here](https://github.com/eventbrite/eventbrite-sdk-javascript/blob/master/CONTRIBUTING.md)
```

Comment:
Can you do:

> Contributions are welcome! See [Contributing Guidelines](CONTRIBUTING.MD) for more details.

Also you can just do `CONTRIBUTING.md` as the URL. You don't have to do the full URL